tests/pytests/test.py

```diff
@@ -4599,8 +4599,16 @@ def test_timeoutCoordSearch_Strict():
     # Search with no timeout limit, get all results
     res = env.cmd('FT.SEARCH', 'idx', '*', 'TIMEOUT', '0')
     env.assertEqual(res[0], n_docs)
-    res = env.cmd('FT.AGGREGATE', 'idx', '*', 'TIMEOUT', '0')
+    # Aggregate with no timeout limit, get all results if WITHCOUNT is set
+    res = env.cmd('FT.AGGREGATE', 'idx', '*', 'WITHCOUNT', 'TIMEOUT', '0')
```

Comment:
The default should be `WITHOUTCOUNT` (implicitly) and it seems we are expecting accurate count of al l docs due to the timeout policy `ON_TIMEOUT FAIL`. Need to understand why not working as before.